docs/r2/tensorboard_in_notebooks.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>We've been calling this and the previous section "Setup", based on DevRel's  guidelines.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1928/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Addressed.